source/manual/covid-19-services.html.md

```diff
@@ -0,0 +1,312 @@
+
+1. Log in with `cf login -a api.cloud.service.gov.uk --sso`
+2. Select staging or production with `cf target -s <staging or production>`
+
+Both applications are Rails applications, and use the
+[`ruby_buildpack`][] to build and deploy to PaaS.  You can read more
+about [managing PaaS applications][paas-managing] and [monitoring PaaS
+applications][paas-monitoring].
+
+#### Manage backing services
+
+The business volunteer application uses PaaS's [Redis backing
+service][paas-redis].  The backing service is attached to the
+application by a setting in the `manifest.yml` file in the repository
+root.
+
+The JSON data in the `VCAP_SERVICES` environment variable exposes
+settings and credentials for the backing service automatically.
+
+### AWS
+
+We have two AWS accounts (staging and production) for CDN and data
+storage. They contain RDS / DynamoDB, IAM and CloudFront resources,
+which were provisioned by Terraform in the [tech-ops-private
+repository](techops-repo).
+
+To log in to AWS:
+
+1. Install `gds-cli` v2.9.0 or above:
+
+   ```
+   brew update && brew upgrade gds-cli
+   ```
+
+2. Log in to the AWS console with either:
+
+   - `gds aws govuk-corona-data-staging -l` for staging
+   - `gds aws govuk-corona-data-prod -l` for production
+
+
+## Deployment
+
+Deployment is managed via [TechOps shared Concourse][big-concourse].
+CI/CD pipelines are configured under the `govuk-tools` team.  Any
+changes to application code on the master branch are continuously
+deployed to staging.  Smoke tests are set up to run on staging, if
+these pass the applications are automatically deployed to production.
+The smoke tests are the feature tests within the application
+repository.
+
+The Concourse deployment pipeline and task configuration for each
+application is stored in the "concourse" directory at the root of each
+repository.
+
+Links to Concourse Pipelines:
+
+- [`govuk-corona-business-volunteer-form`](https://cd.gds-reliability.engineering/teams/govuk-tools/pipelines/govuk-corona-business-volunteer-form)
+- [`govuk-corona-vulnerable-people-form`](https://cd.gds-reliability.engineering/teams/govuk-tools/pipelines/govuk-corona-vulnerable-people-form)
+
+For access login with your GitHub account, you need to be a part of
+the GOV.UK Production team.  You'll also need to be on the VPN.
+
+### Administering Pipelines
+
+To administer pipelines you must have the `pipeline-operator` role.
+Pipelines are administered using the `fly` CLI tool.  `fly` can be
+downloaded directly from the Concourse dashboard using the links in
+the bottom bottom right of the screen (you probably want to click the
+apple).
+
+To get started, firstly configure a target and log in:
+
+```
+fly --target cd-govuk-tools login --team-name govuk-tools --concourse-url https://cd.gds-reliability.engineering
+```
+
+Changes to the pipeline have to be applied manually using the
+`set-pipeline` command.  To apply changes, run the following command
+from the project root:
```

Comment:
Maybe add a line "Committing the changes to master will not update the pipeline." to be clear?